### ISSUE TITLE
[BUG] Watering Can gives Exp mod

### ISSUE THREAD

**Issue by Chargoal** [Achtuur/StardewMods#2]
Mod: Watering Can gives Exp
TL;DR: weird Generic Mod Config Menu interaction when setting "Exp gain per tile" to 0,5.

Setup/Way to recreate: I have UI Suite 2 installed and it shows me every time I gain an experience point. I didn't change any default setting of your mod at first. I saw that it was set to 0,25 in GMCM, which I was okay with, but in the game it gave me 1 exp per 2 TILES, as if I had 0,5 set.

I dug deeper and saw that the config file actually has the number set to 0,5. I tried changing it to other values in GMCM and everything worked fine, but whenever I set it to 0,5 and saved it, it then after re-opening the menu showed me that my config is set to 0,25 which was not true practically and in the config file itself.
It seems that there is weirdness in GMCM reflecting config file value when set to 0,5. I have no idea why.
